frontend/src/i18n/locales/en.yaml

```diff
@@ -17,7 +17,7 @@ general:
     goto-homepage: Go to homepage
     share: Share
   version-information: Version
-  logo-alt: Logo of “{{title}}”
+  logo-alt: Link to “{{title}}”
```

Comment:
Hu ok, I googled a bunch and learned something. So most sources indeed say that the alt text of a linked image should not describe the image, but the action of the link. Which is weird to me but fine. [This source](https://www.mtu.edu/accessibility/training/web/link-text/) says "The ALT tag must include both the content and function of the image. If the image is decorative, then only function is necessary." Is the logo decorative? Maybe, and anyway, again, most other sources say "only describe the function".

[This page](https://sc.edu/about/offices_and_divisions/digital-accessibility/toolbox/best_practices/alternative_text/linked-images/index.php) says "Using phrases like 'Link to...' or 'Click to...' in your alt text is redundant for screen reader users.". My screen reader reads "${alt_text}. Visited link". So I think I would change this to "Go to homepage of “{{title}}”" or just "Homepage of “{{title}}”". What do you think?

Reply:
Yup that sounds reasonable.